webserver.py

```diff
@@ -116,6 +125,33 @@ def determine_color(uk_time):
         return STATUS_OUT
 
 
+def get_train_details():
+    if not TRANSPORT_APP_KEY or not TRANSPORT_APP_ID or not FAVOURITE_STATION:
```

Comment:
This:
```suggestion
    if not (TRANSPORT_APP_KEY and TRANSPORT_APP_ID and FAVOURITE_STATION):
```
might read better